airflow/bin/cli.py

```diff
@@ -707,6 +707,9 @@ class CLIFactory(object):
         'conf': Arg(
             ('-c', '--conf'),
             "json string that gets pickled into the DagRun's conf attribute"),
+        'date': Arg(
```

Comment:
The execution_date arg creates a required parameter, i.e. airflow run dag_id task_id execution_date. Maybe a better name for the arg could be found, but I think this should be an optional parameter.